docs/fundamentals/code-analysis/style-rules/ide0003_ide0009.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "'this.' and 'Me.' qualifiers (IDE0003 and IDE0009)"
+description: "Learn about code analysis rules IDE0003 and IDE0009: 'this.' and 'Me.' qualifiers"
+ms.date: 09/30/2020
+ms.topic: reference
+f1_keywords:
+- IDE0003
+- IDE0009
+- dotnet_style_qualification_for_field
+- dotnet_style_qualification_for_property
+- dotnet_style_qualification_for_method
+- dotnet_style_qualification_for_event
+helpviewer_keywords:
+- IDE0003
+- IDE0009
+- dotnet_style_qualification_for_field
+- dotnet_style_qualification_for_property
+- dotnet_style_qualification_for_method
+- dotnet_style_qualification_for_event
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# 'this.' and 'Me.' qualifiers (IDE0003 and IDE0009)
+
+|Property|Value|
+|-|-|
+| **Rule ID** | IDE0003 and IDE0009 |
+| **Title** | IDE0003: Remove 'this' or 'Me' qualification<br/> IDE0009: Add 'this' or 'Me' qualification |
```

Comment:
```suggestion
| **Title** | IDE0003: Remove `this` or `Me` qualification<br/>IDE0009: Add `this` or `Me` qualification |
```

Not sure if the quotes are supposed to match the exact message that's emitted. If so, you can ignore this.